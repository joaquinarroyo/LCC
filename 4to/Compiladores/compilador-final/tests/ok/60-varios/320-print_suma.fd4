let _ : Nat =
  (print "uno" 1 + print "dos" 2) +
  (print "tre" 3 + print "cua" 4)
