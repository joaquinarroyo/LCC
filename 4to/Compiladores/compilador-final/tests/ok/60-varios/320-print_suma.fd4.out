uno1
dos2
tre3
cua4
